ci/cloudbuild/triggers/demo-centos-7-ci.yaml

```diff
@@ -0,0 +1,13 @@
+filename: ci/cloudbuild/cloudbuild.yaml
```

Comment:
Q: I assume you cannot enable these triggers until the code is merged, right?

Reply:
Correct. Well, I _could_ enable them and they would work on _my_ PR because my PR has the right scripts. However, if someone else submitted a PR, the trigger would fail for them, so I tend not to enable triggers until after I merge the required scripts.